powerlevel9k.zsh-theme

```diff
@@ -955,6 +1007,12 @@ function zle-keymap-select {
   zle reset-prompt
 }
 
+function showWarningIfDeprecated() {
```

Comment:
Just for consistency's sake, maybe this should be named `show_warning_if_deprecated` to avoid mixing camel and snake casing?